akka-sample-distributed-workers-scala/README.md

```diff
@@ -1,52 +1,52 @@
 # Akka Distributed Workers with Scala Guide
 
-To be reactive, distributed applications must deal gracefully with temporary and prolonged outages as well as have 
-the ability to scale up and down to make the best use of resources. 
-Akka Cluster provides these capabilities so that you don't have to implement them yourself. 
+To be reactive, distributed applications must deal gracefully with temporary and prolonged outages as well as have
```

Comment:
Visual Code was showing lots of Markdown warnings. So I removed them all. Nothing important.